src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,202 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = LCDFeatureSet( )
+            obj = obj@FeatureCreators.Base();
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...

```

Comment:
Localisation needs `pp_bBinauralRMS` set to true. Identification and joint id+loc doesn't care.
We will keep 'pp_bNormalizeRMS' set to true. No bad effects for anyone.
`pp_bBinauralRMS` will be set to `true`. That way we can reduce the ratemaps requests to 1 request for all.